src/components/bcros/filing/CommonTemplate.vue

```diff
@@ -77,6 +77,16 @@
         </template>
       </slot>
 
+      <slot name="document-record">
+        <!-- if we have document records(aka filings staff completed on behalf of a client), show them -->
+        <template
+          v-if="!isStaffFiling(filing) && enableDocumentRecords && !!getDocIdByFilingId(documents, filing.filingId)"
```

Comment:
Not staff filing? I think Court Order might be an exception. (It's currently a "simple dialog" staff filing but it's being moved to a one-page filing like the others.)

Reply:
Good question, is Court Order a filing that Staff would do on behalf of a client?
ie a Client has either mailed/emailed/brought in documentation that would than require staff to do on their behalf

Reply:
Yes.

Reply:
Just to follow up on this, are any other of these Staff Filings also completed at the behest of the client?

<img width="271" alt="Screenshot 2025-01-28 at 9 49 38 AM" src="https://github.com/user-attachments/assets/aa59d4cb-a225-4346-86b1-7e244e90f52d" />

Reply:
I think Court Order is the only one.

Reply:
Ok, thanks for this info sir! 

We currently support Court Order documents on the DRS side i believe, although its not classed as a Corp specific filing (which makes sense). 
Are you comfortable with me collecting a bit more info/requirements from our BA's before enabling on this end?

Reply:
> Are you comfortable with me collecting a bit more info/requirements from our BA's before enabling on this end?

What do you mean, specifically, by "enabling"?

Reply:
Oh, you mean just turning on the FF in Dev and maybe Test?

When do you plan to enable this in Prod?

Reply:
I mean by implementing a condition in the logic to allow it for Court Orders, i just want to follow up on scope, requirements and support on our end.

Reply:
OK, but be advised we're currently working on the stand-alone Court Order filing. It will be just like the other stand-alone filings in Filings UI (except staff-only).

Thanks for the follow-up.

Reply:
Thanks sir, i will follow up on this. 
Saving these docs to the DRS vs Minio will happen eventually but you raise a good question.

 If this filing is done by staff due to a received paper form which they have need to apply a barcode to before scanning, it would most likely require a Document Id capture component. 
Will follow up with the BA's 100%!

Reply:
Zactly.

What about the other questions here:
- do you need the FF check or is it redundant?
- do you need the staff filing check or is it redundant?

Reply:
I think we will want the FF yes but i'm going to refactor to request on panel expand. We will only need it in that one place. 
The staff filing condition is probably redundant, was more of cautious coding but i think the way it's done now, we won't rending in absence of filingId === consumerReferenceId match, so it's probably safe to remove.